client/components/App.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
```

Comment:
I think this would work without the need to make these files into `.jsx` only because we have a babel compiler than converts ES6. 

But I think this is okay.